File: frontend/src/app/api/models/home-summary-dto.ts
/* tslint:disable */
import { BudgetSummaryDto } from './budget-summary-dto';
import { CalendarSummaryDto } from './calendar-summary-dto';
import { GuestSummaryDto } from './guest-summary-dto';
export interface HomeSummaryDto {
  budgetSummary?: BudgetSummaryDto;
  calendarSummary?: CalendarSummaryDto;
  ceremonyDate: string;
  ceremonyPlace?: string;
  guestSummary?: GuestSummaryDto;
  isWeddingDetailsSaved: boolean;
  partnerName?: string;
  userName?: string;
  weddingDate: string;
  weddingPlace?: string;
}

File: WPA.backend/WPA.backend/DTOs/Home/HomeSummaryDto.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WPA.backend.DTOs.Home
{
    public class HomeSummaryDto
    {
        public string UserName { get; set; }
        public string PartnerName { get; set; }
        public DateTime WeddingDate { get; set; }
        public string WeddingPlace { get; set; }
        public DateTime CeremonyDate { get; set; }
        public string CeremonyPlace { get; set; }
        public bool IsWeddingDetailsSaved { get; set; }
        public virtual GuestSummaryDto GuestSummary { get; set; }
        public virtual BudgetSummaryDto BudgetSummary { get; set; }
        public virtual CalendarSummaryDto CalendarSummary { get; set; }
    }
}
